driver/src/main/java/org/neo4j/driver/internal/cluster/RediscoveryImpl.java

```diff
@@ -53,13 +53,15 @@
 import static org.neo4j.driver.internal.util.Futures.failedFuture;
 
 /**
- * This class is used by all router tables to perform discovery.
- * In other words, the methods in this class could be called by multiple threads concurrently.
+ * This class is used by all router tables to perform discovery. In other words, the methods in this class could be called by multiple threads concurrently.
```

Comment:
Normally, I wouldn't complain, but as you touched it: Why not make this say: "The methods of this class must be thread safe as an instance of this is used by all router tables concurrently.."

Reply:
I actually did not intentionally change it, the formatter did and I did not mind it formatting this block according to its rules.

Reply:
But I can update this comment if you wish.